tests/kafkatest/tests/streams/streams_broker_compatibility_test.py

```diff
@@ -79,11 +88,11 @@ def test_fail_fast_on_incompatible_brokers_if_eos_enabled(self, broker_version):
     @parametrize(broker_version=str(LATEST_1_1))
     @parametrize(broker_version=str(LATEST_1_0))
     @parametrize(broker_version=str(LATEST_0_11_0))
-    def test_compatible_brokers_eos_disabled(self, broker_version):
+    def test_compatible_brokers_eos_enabled(self, broker_version):
```

Comment:
Add this new test as side cleanup -- not sure why we never tested EOS for older broker but only non-EOS?